ui/app/mainui/AppMain.qml

```diff
@@ -1342,7 +1344,8 @@ Item {
     DropAreaPanel {
         width: appMain.width
         height: appMain.height
-        activeChatType: chatCommunitySectionModule && chatCommunitySectionModule.activeItem.type
+        activeChatType: rootStore.chatCommunitySectionModule ? rootStore.chatCommunitySectionModule.activeItem.type || 0
```

Comment:
I believe this is wrong... `rootStore.chatCommunitySectionModule` doesn't exist in this context... dunno where it's actually coming from :)

Reply:
Updated